requirements/requirements.txt

```diff
@@ -10,3 +10,4 @@ pandas~=1.0
 pydantic~=1.1
 tqdm~=4.23
 ujson~=1.35
+statsmodels~=0.11
```

Comment:
This seems to be quite a big dependency. Can we move this to an extra?

Reply:
yes, good idea

Reply:
It doesn't seem to have any conflicts when I installed it, and it has pretty basic dependencies.

```
Python >= 3.5
NumPy >= 1.14 # we require the same
SciPy >= 1.0
Pandas >= 0.21 # we require 1.0
Patsy >= 0.5.1
```

Additionally, not having it in there would make using OWA without extra not possible.
So, overall, I dislike the idea.

Reply:
I agree it's better to have it in extra

Reply:
I guess we could put it into `requirements-extras-m-competition.py` if necessary.